cosi.proto

```diff
@@ -126,7 +170,7 @@ message ProvisionerDeleteBucketRequest {
     
     // This field is OPTIONAL
     // Protocol specific information required by the call is passed in as key,value pairs.
-    map<string,string> bucket_context = 2;
+    map<string,string> parameters = 2;
```

Comment:
Does deletion really require opaque parameters?

Reply:
good catch. It doesn't

Reply:
Thinking about it more, yes it does. If there are driver specific parameters required to address the bucket, this is the only handle we have

Reply:
Do you have concrete cases of that? In general for CSI we've tried to make deletion as minimal as possible (going as far as not allowing any secrets on the delete call) so that we minimize the possibility of CO ending up in a state where it can't clean up.